workflows/r/untitled.txt

```diff
@@ -0,0 +1,107 @@
+# Import packages that you need 
+library(dplyr)
+library(odbc)
+library(DBI)
+library(tidymodels)
+
+# Setting some variables
+bucket = Sys.getenv('DATA_LAKE') # For the data lake connection
+filepath = paste0(Sys.getenv('DATA_LAKE_ROOT_PATH') , 'datascience/') # For the data lake connection
+modelname = 'model.rds' # Model name
+
+# Connection string for Snowflake
+print("Connecting to Snowflake")
+conn = dbConnect(odbc::odbc(),
+                 driver = '/usr/lib/snowflake/odbc/lib/libSnowflake.so',
+                 server = paste0('peak-', Sys.getenv('TENANT'),'.snowflakecomputing.com'),
+                 database = Sys.getenv('SNOWFLAKE_DATABASE'),
+                 uid = Sys.getenv('SNOWFLAKE_USERNAME'),
+                 pwd = Sys.getenv('SNOWFLAKE_PASSWORD')
+)
+
+# Query the raw data and filter out house prices 
+print("Get data and filter price")
+df = dbGetQuery(conn, "select * from STAGE.HOUSEPRICES") %>%
+  filter(price > 0)
+
+# Removing categorical variables
+print("Only select numerical data")
+data = df %>%
+  select(-date,
+         -street, 
+         -city, 
+         -statezip,
+         -country, 
+         -PEAKAUDITCREATEDAT, 
+         -REQUEST_ID)
+
+# Building and fitting Linear Regression model. You can change this
+print("Build model")
+linreg_reg_spec <- 
+  linear_reg() %>% 
+  set_engine("lm")
+
+linreg_reg_fit <- linreg_reg_spec %>% fit(price ~ ., data = data)
+
+# Predicting on our data and creating a column called predictions
```

Comment:
Hey @vanessavirgo94 - Do we want to keep the file name as untitled ? Seems a bit odd to me.